docs/content/docs/user-docs/openshift.md

```diff
@@ -109,7 +109,22 @@ Delete `config.txt` and `secrets.txt`.
 If you change the name of either the `ConfigMap` or the `Secret` from the values given above, i.e. `ack-$SERVICE-user-config` and `ack-$SERVICE-user-secrets`, then installations from OperatorHub will not function properly. The Deployment for the controller is preconfigured for these key values.
 {{% /hint %}}
 
-### Step 4: Install the controller
+### Step 4 (Optional): Apply Additional Custom Resource Definitions(CRD)
+CRDs are global resources, they cannot be shared, or installed by multiple controllers. This is why [AdoptedResource](../../../reference/common/v1alpha1/adoptedresource/) and [FieldExport](../../../reference/common/v1alpha1/fieldexport/)
```

Comment:
I might reword/reorder this to something like:

"To prevent CRD installation conflicts for CRDs shared across multiple AWS Controllers for Kubernetes, the AdoptedResource and FieldExport CRDs are not included in the OpenShift Embedded Operator Hub. These must be installed manually by a cluster administrator before any controller is installed by running the following commands":

Reply:
That's alot better then what I had. Thanks.